examples/nextjs-swc/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev --turbopack",
```

Comment:
should this be --turbo?

Reply:
This was done automatically by theirs migration script

Reply:
You are right, they have just updated their docs 👍

And it was recent enough that google hadn't indexed the change yet when I wrote the comment. Sorry for the nitpick 🙂